views/includes/page-templates/trust-involved.html

```diff
@@ -0,0 +1,205 @@
+{% extends "layout.html" %}
+
+{% set title = pageParams.title %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
```

Comment:
only the stuff inside block content should exist inside this shared page-template file.
Then in the 2 views that include this one you need to have extends, title, backlink etc that follows something like done here: https://github.com/companieshouse/overseas-entities-web/pull/975/files#diff-903ea6498b54fc034a7ea4d7508dfc1144eb8fc880535cba4ea792a1241408b0L1

Because these things differ between registration and update joourneys.

Reply:
Fixed